.docker/qgis/scripts/qgis_testrunner.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This file you don't need to add, is already in the docker image: /usr/bin/qgis_testrunner.sh

Reply:
removed